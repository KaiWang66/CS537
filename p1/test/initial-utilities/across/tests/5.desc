no filename, check special charactor
