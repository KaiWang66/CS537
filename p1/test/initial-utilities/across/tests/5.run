./across $(< tests/5.target)
