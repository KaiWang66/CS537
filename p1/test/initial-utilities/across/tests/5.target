la 1 7