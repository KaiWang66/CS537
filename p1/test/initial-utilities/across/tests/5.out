alarmed
blabbed
blacked
blacken
blacker
bladder
blaming
blander
blandly
blanked
blanker
blanket
blankly
blaring
blarney
blasted
blaster
blatant
blazers
blazing
blazons
clacked
claimed
clamber
clammed
clamors
clamped
clanged
clangor
clanked
clapped
clapper
clarets
clarify
clarion
clarity
clashed
clashes
clasped
classed
classes
classic
clatter
clauses
clawing
clayier
elapsed
elapses
elastic
elating
elation
flaccid
flagged
flagons
flailed
flakier
flaking
flambes
flaming
flanges
flanked
flannel
flapped
flapper
flaring
flashed
flasher
flashes
flatbed
flatcar
flatted
flatten
flatter
flattop
flaunts
flavors
flawing
flaying
glacial
glacier
gladden
gladder
glamors
glamour
glanced
glances
glaring
glassed
glasses
glazier
glazing
placard
placate
placebo
placers
placing
placket
plagued
plagues
plainer
plainly
plaints
plaited
planets
planing
planked
planned
planner
planted
planter
plaques
plaster
plastic
plateau
platens
plating
platoon
platter
platypi
plaudit
playact
playboy
players
playful
playing
playoff
playpen
slabbed
slacked
slacken
slacker
slackly
slaking
slaloms
slammed
slammer
slander
slanted
slapped
slashed
slashes
slather
slating
slavers
slavery
slaving
slavish
slayers
slaying
